NickvisionMoney.GNOME/Views/TransactionDialog.cs

```diff
@@ -245,12 +246,36 @@ private TransactionDialog(Gtk.Builder builder, TransactionDialogController contr
         {
             if(e.Flags.HasFlag(Gtk.StateFlags.FocusWithin) && !_descriptionRow.GetStateFlags().HasFlag(Gtk.StateFlags.FocusWithin))
             {
+                _descriptionRow.SetActivatesDefault(true);
                 _autocompleteBox.SetVisible(false);
             }
         };
         _descriptionKeyController = Gtk.EventControllerKey.New();
         _descriptionKeyController.SetPropagationPhase(Gtk.PropagationPhase.Capture);
-        _descriptionKeyController.OnKeyPressed += (sender, e) => { if (e.Keyval == 59) { return true; } return false; };
+        _descriptionKeyController.OnKeyPressed += (sender, e) =>
+        {
+            if (e.Keyval == 59) //semicolon
+            {
+                return true;
+            }
+            if(e.Keyval == 65293 || e.Keyval == 65421) //enter | keypad enter
+            {
+                if(_autocompleteBox.GetVisible())
+                {
+                    _autocompleteBox.AcceptSuggestion(0);
+                    return true;
+                }
+            }
+            if(e.Keyval == 65364) //down arrow
+            {
+                if(_autocompleteBox.GetVisible())
+                {
+                    _autocompleteBox.GrabFocus();
+                    return true;
+                }
+            }
```

Comment:
```suggestion
            if(_autocompleteBox.GetVisible())
            {
                if(e.Keyval == 65293 || e.Keyval == 65421) //enter | keypad enter
                {
                    _autocompleteBox.AcceptSuggestion(0);
                    return true;
                }
                if(e.Keyval == 65364) //down arrow
                {
                    _autocompleteBox.GrabFocus();
                    return true;
                }
            }
```

Reply:
I don't want to do it like this incase we need to keys for something else and need to add more if statements. Best to keep the outside if statements just comparing keyvals as it is a OnKeyPressed handler.